data/sql/updates/pending_db_world/rev_1595112699414325989.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1595112699414325989');
+
+UPDATE `gameobject` SET `spawntimesecs`=2 WHERE guid IN (36738, 182011, 142088, 176484, 176485, 176486, 176487);
+UPDATE `gameobject` SET `spawntimesecs`=180 WHERE guid IN (182199, 176249, 177287);
+UPDATE `gameobject` SET `spawntimesecs`=300 WHERE guid IN (20726, 19284, 19283, 103664, 142477, 13949, 177964, 91138, 113757, 125477);
+UPDATE `gameobject` SET `spawntimesecs`=7200 WHERE guid = 179545;
+
```

Comment:
you forgot the book of ur aka 40667

Reply:
The guid `40667` has already a low spawntimesecs (2), I don't think is necessary to put it as 3.